frontend/src/routes/_dashboard/create-project.lazy.tsx

```diff
@@ -28,11 +35,50 @@ function AuthedView() {
 }
 
 export function CreateProject() {
+  const navigate = useNavigate();
+
+  const { isPending, mutate, data } = useMutation({
+    mutationFn: async (input: RequestInput<"post", "/api/projects/">) => {
+      await apiClient.GET("/api/csrf/");
+      const csrfToken = getCookie("csrftoken");
```

Comment:
It would be interesting to abstract this into its own function because it will be used everywhere.